openlibrary/templates/account/mybooks.html

```diff
@@ -0,0 +1,52 @@
+$def with (user, mybooks, public=False, owners_page=False, counts=None, lists=None)
+
+$ component_times = {}
+$ component_times['TotalTime'] = time()
+
```

Comment:
```suggestion
```
Profiling isn't happening on this page.  If there is a need for it, we can either pass `component_times` to this template from `/account.books.html`, or make `component_times` global.